theforeman.org/yaml/jobs/packaging/pulpcore-packaging-rpm-pr-test.yaml

```diff
@@ -18,3 +18,23 @@
         - pipelines/lib/git.groovy
         - pipelines/lib/ansible.groovy
         - pipelines/lib/obal.groovy
+
+- job:
+    name: pulpcore-packaging-rpm-copr-pr-test
+    project-type: pipeline
+    concurrent: true
+    sandbox: true
+    properties:
+      - github:
+          url: https://github.com/theforeman/pulpcore-packaging
+    triggers:
+      - github_pr_rpm_develop:
```

Comment:
This is copied from the foreman job where we are restricting PR testing to only rpm/develop branch. What branches do you want to see this in for Pulpcore?